modules/isIterable.js

```diff
@@ -0,0 +1,7 @@
+function isIterable(obj) {
+    // checks for null and undefined
+    if (obj == null) {
+      return false;
+    }
+    return typeof obj[Symbol.iterator] === 'function';
```

Comment:
Given my previous comment, the entire body of this function would actually fit into a single expression:

```suggestion
    return obj != null && isFunction(getIterator(obj));
```

Though `getIterator` should probably check against `null` and `undefined` anyway, so you could actually leave out the null check entirely.